OpenRA.Mods.Common/UtilityCommands/OutputActorMiniYamlCommand.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Linq;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	public class OutputActorMiniYamlCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name { get { return "--actor-yaml"; } }
+
+		bool IUtilityCommand.ValidateArguments(string[] args)
+		{
+			return args.Length == 2 || args.Length == 3;
+		}
+
+		[Desc("ACTOR-TYPE [PATH/TO/MAP]", "Display the merged MiniYaml tree for the given actor type. Input values are case-sensitive.")]
+		void IUtilityCommand.Run(ModData modData, string[] args)
+		{
+			// HACK: The engine code assumes that Game.modData is set.
+			Game.ModData = modData;
+
+			var actorType = args[1];
+			var mapPath = args[2];
```

Comment:
I think this is bogus.

If `Length == 2`:
[0] will be the command
[1] the actor type
[2] **exception**

if `Length == 3`:
[0] will be the command
[1] the actor type
[2] the map path

Will fix.